src/components/header/index.js

```diff
@@ -28,13 +28,13 @@ export const Header = () => {
 
           <div className={styles.right}>
             <Button onClick={context.syncTaquito} secondary>
-              <Secondary>sync</Secondary>
+              <Secondary className={styles.textcolor}>sync</Secondary>
```

Comment:
I'm not a fan of this API. the styling for the <Secondary /> button should be on the sass file of the button. src/components/button/index.module.scss The idea is to just import components and not worrying about how that variable gets defined.

Reply:
So the idea with <Secondary> is that it takes care of the layout for you. Do not pass classes into it. The button itself reads the variables.